lnd.go

```diff
@@ -226,11 +228,19 @@ func Main(lisCfg ListenerCfg) error {
 	ltndLog.Infof("Opening the main database, this might take a few " +
 		"minutes...")
 
+	cfg.DB.Path = graphDir // TODO(bhandras)
```

Comment:
This should likely go before this is merged. The path from the config should be used.

Reply:
I think the config must be almost parsed to fill the path, so rather I created a function in `config.go` to return the local database directory. This is then passed as a param to the `GetBackend` function. We could of course expose database path in the config, but I'm not sure if we should? Maybe this is something our users want though.